pkg/apis/core/types_shoot.go

```diff
@@ -335,6 +335,8 @@ type ClusterAutoscaler struct {
 	MaxNodeProvisionTime *metav1.Duration
 	// MaxGracefulTerminationSeconds is the number of seconds CA waits for pod termination when trying to scale down a node (default: 600).
 	MaxGracefulTerminationSeconds *int32
+	// IgnoreTaint specifies a taint to ignore in node templates when considering to scale a node group
+	IgnoreTaint []string
```

Comment:
plural?

Reply:
Even I felt like this should be plural but the flag in CA [here[(https://github.com/gardener/autoscaler/blob/2bf7dbe0a9dfe0a49f6c31186d17927cd0e3df26/cluster-autoscaler/main.go#L172) is in singular form, so I thought of maintaining the consistency. 

But probably, plural makes more sense from Gardener perspective. I will adapt this one.